### DISCUSSION TITLE
Using s3 compliant storage in Google Cloud

### DISCUSSION THREAD

**Discussion by moficodes** [felddy/foundryvtt-docker#982] [Q&A]
This is 2 part questions.

1. According to [google clouds doc](https://cloud.google.com/storage/docs/samples/storage-s3-sdk-list-buckets) https://storage.googleapis.com is the endpoint for gcs. But when I used it in my awsConfig.json file got a 
```
Failed to determine S3 endpoint: Invalid argument.
```

Am I doing something wrong?

Complete `awsConfig.json`
```
{
  "region": "us-east-1",
  "endpoint": "https://storage.googleapis.com",
  "credentials": {
    "accessKeyId": "KEY",
    "secretAccessKey": "MY ACCESS KEY"
  }
}
```

2. How does foundry know which bucket to use? How do I specify a bucket in configuration?

If anyone can point me towards some direction that would be very kind.

**Comment by felddy** [comment]
I haven't personally used Foundry's AWS integration but the official documentation is here:
- https://foundryvtt.com/article/aws-s3/

The configuration can be passed to the container via the `FOUNDRY_AWS_CONFIG` [environment variable](https://github.com/felddy/foundryvtt-docker?tab=readme-ov-file#optional-variables). 

See also:
- https://github.com/felddy/foundryvtt-docker/discussions/328
